pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -107,111 +108,135 @@ def connection_delete(context, name, **options):
-    if NAME exists in the store. The NAME argument is optional.  If NAME not
-    supplied, a list of connections from the connections definition file
-    is presented with a prompt for the user to select a NAME.
+    if NAME exists in the store. The NAME argument is
+    optional. If NAME not supplied, a list of connections from the
+    connections definition file is presented with a prompt for the user to
+    select a connection.
 
     Select state is not persistent.
 
     Examples:
 
-       connection select <name>    # select the defined <name>
+       connection select myconn    # select the connection named 'myconn'
 
        connection select           # presents select list to pick connection
     """
 
     context.execute_cmd(lambda: cmd_connection_select(context, name))
 
 
-# pylint: disable=bad-continuation
 @connection_group.command('add', options_metavar=CMD_OPTS_TXT)
-@click.option('-s', '--server', type=str, metavar='SERVER', required=False,
-              help='Required hostname or IP address with scheme of the '
-                   'WBEM server in format:\n[{scheme}://]{host}[:{port}]\n'
-                   '* Scheme: must be "https" or "http" [Default: "https"]\n'
-                   '* Host: defines short/fully qualified DNS hostname, '
-                   'literal IPV4 address (dotted), or literal IPV6 address\n'
-                   '* Port: (optional) defines WBEM server port to be used '
-                   '[Defaults: 5988(HTTP) and 5989(HTTPS)].\n')
-@click.option('-n', '--name', type=str, metavar='NAME', required=True,
-              help='Required name for the connection(optional, see --server).  '
-                   'This is the name for this defined WBEM server in the'
-                   ' connections file')
-@click.option('-d', '--default-namespace', type=str,
-              metavar='NAMESPACE',
-              default=DEFAULT_NAMESPACE,
-              help="Default namespace to use in the target WBEM server if no "
-                   "namespace is defined in the command"
-                   " (Default: {of}).".format(of=DEFAULT_NAMESPACE))
-@click.option('-u', '--user', type=str,
-              help="User name for the WBEM server connection. ")
-@click.option('-p', '--password', type=str,
-              envvar=PywbemServer.password_envvar,
-              help="Password for the WBEM server. Will be requested as part "
-                   " of initialization if user name exists and it is not "
-                   " provided by this option.")
-@click.option('-t', '--timeout', type=click.IntRange(0, MAX_TIMEOUT),
-              help="Operation timeout for the WBEM server in seconds. "
-                   "Default: " + "%s" % DEFAULT_CONNECTION_TIMEOUT)
+@click.option('-n', '--name', type=str, metavar='NAME',
+              required=True,
+              help='Name of the persistent WBEM connection to be added.')
+@click.option('-m', '--mock-server', type=str, multiple=True, metavar="FILE",
```

Comment:
Most of the remaining options apply to --server. That's why I wanted to have them adjacent to --server. In addition, I wanted to have --mock-server and --server adjacent to each other because they define the main choice.

Reply:
Lets keep andys order.